repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1977,6 +2060,12 @@ private void flushAutoUpdateAttributes(AtlasEntity entity, AtlasEntityType entit
                 }
             }
 
+            for (String attrName : entityType.getAllAttributes().keySet()) {
+                if (ATTR_STARRED_BY.equals(attrName) || ATTR_STARRED_COUNT.equals(attrName) || ATTR_STARRED_DETAILS_LIST.equals(attrName)) {
+                    flushAttributes.add(attrName);
+                }
+            }
+
```

Comment:
Already iterating on entityType.getAllAttributes().keySet(), check if we can move this intp previous for loop

Reply:
Commenting this logic for now to ensure backward compatibility. Will add it once frontend changes are released and will see if we can leverage the same loop.